src/SbManager/Scripts/manager/statusController.js

```diff
@@ -0,0 +1,20 @@
+﻿$app.controller('statusController', ['$scope', function ($scope) {
+    $scope.refresh = function () {
+        $scope.model = null;
+        $.ajax({
+            url: window.applicationBasePath + "/api/v1/busmanager/",
+            dataType: 'json',
+            success: function (d) {
```

Comment:
Success and error callbacks are deprecated in angular and removed in a later version. Can we please use promises here as we would like to upgrade the angular version as some stage.